README.md

```diff
@@ -1,9 +1,25 @@
 # Support
 
 Forms that create Zendesk tickets for requests coming from government agencies.
-The app is written in Rails and uses the zendesk_api gem to connect to Zendesk.
 
-### Getting started
+## Technical documentation
+
+This is a Ruby on Rails application which is not public facing. However, it does face GOV.UK employees and is used as a way to submit Zendesk tickets. 
```

Comment:
GOV.UK employees isn't quite right.  It's available to all signon users, most of whom don't even work for GDS, let alone work on GOV.UK.  I think it's being opened up to local government shortly too.